### DISCUSSION TITLE
[Deploy to Vehicle] - Blacklist vehicles via setVariable?

### DISCUSSION THREAD

**Discussion by OverlordZorn** [Vdauphin/HeartsAndMinds#1517] [General]
Hi

For my current version of HAM I encountered a situation that would need me to put multiple vehicles into the btc_respawnable vehicle layer. As far as I know, all vehicles (cars/heli etc depending on the param) will be potential destination for players to deploy to.

Is there a way to blacklist a vehicle so these specific objects do not show up as destination via the Deploy Option?

**Comment by Vdauphin** [comment]
Hello,
You can disable respawn to helo with mission parameters:
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L80-L85
Cheers
